newIDE/app/src/ProjectsStorage/ProjectContentChecker.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import { showErrorBox } from '../UI/Messages/MessageBox';
+import { t } from '@lingui/macro';
+import { I18n } from '@lingui/react';
+import { type I18n as I18nType } from '@lingui/core';
+
+const gd = global.gd;
+
+export default function verifyProjectContent(i18n, content) {
+  const serializedProject = gd.Serializer.fromJSObject(content);
```

Comment:
You don't need to use fromJSObject here: it's used to transform "content" (which is a JSON parsed object) to a gd.SerializerElement (a representation of JSON in memory in libGD.js).
So you can remove this line entirely.